examples/ldap/README.md

```diff
@@ -1,4 +1,4 @@
-[base]: ../../docs/glossary.md#base
+[base]: ../../docs/Reference/glossary.md#base
```

Comment:
I opened the link and got a `404`. Did you want to refer to either of the links below?
- https://github.com/kubernetes-sigs/kustomize/blob/master/site/content/en/docs/Reference/API/Kustomization%20File/bases.md
- https://kubectl.docs.kubernetes.io/references/kustomize/glossary/#base

Reply:
corrected multiple links and pushed again